resources/views/components/button/index.blade.php

```diff
@@ -1,6 +1,7 @@
 <component
     is="{{ $component ?? ($attributes->has('href') || $attributes->has('v-bind:href') ? 'a' : 'button') }}"
-    {{ $attributes->class('text-sm inline-block text-center py-4 px-6 rounded self-start') }}
+    {{ $attributes->class('inline-block self-start rounded py-4 px-6 text-center text-sm transition disabled:cursor-not-allowed disabled:opacity-70') }}
```

Comment:
It does nothing in practice but might be nice for readability
```suggestion
    {{ $attributes->class([
        'inline-block self-start rounded py-4 px-6 text-center text-sm transition',
        'disabled:cursor-not-allowed disabled:opacity-70',
    ]) }}
```
separating disabled state styling into a new array item so it's visually separated from the default state